core/feed/annotator/base.py

```diff
@@ -93,8 +93,8 @@ def contributor(
             return None
 
         name = contributor.display_name or contributor.sort_name
-        name_key = name.lower()
-        if name_key in state[marc_role]:
+        name_key = name and name.lower()
+        if not name_key or name_key in state[marc_role]:
             # We've already credited this person with this
```

Comment:
Is this comment still accurate with this change? Seems like it needs an extra note about empty author.